src/@utils/accessDetailsAndPricing.ts

```diff
@@ -16,7 +16,7 @@ import { AccessDetails, OrderPriceAndFees } from 'src/@types/Price'
 import Decimal from 'decimal.js'
 import { consumeMarketOrderFee } from '../../app.config'
 
-const TokensPriceQuery = gql`
+const TokensPriceQueryOp = gql`
```

Comment:
what was the reason for renaming those?

Reply:
at same point we have an error when compiling because it was named the same as the import in [line 8](https://github.com/oceanprotocol/market/blob/da7ccf392add5ad996e4dba16873e2f409215bdb/src/%40utils/accessDetailsAndPricing.ts#L8)

Reply:
alright, makes sense, wanted to find a way to not touch this file to get the CodeClimate errors away which are all reported cause we touched this file here